draft-ietf-quic-transport.md

```diff
@@ -6369,14 +6369,17 @@ endpoints that share a static key for stateless reset (see {{reset-token}}) MUST
 be arranged so that packets with a given connection ID always arrive at an
 instance that has connection state, unless that connection is no longer active.
 
+More generally, servers MUST NOT generate a stateless reset if a connection with
+the corresponding connection ID could be active.
```

Comment:
```suggestion
the corresponding connection ID could be active on any endpoint using the same static key.
```
Obviously this isn't intended to be a global statement, so let's make the scope clear.

Reply:
Well, I consider this to be implicit.  After all, your deployment can mint the same connection ID as mine and they would be different.  But it's a fine clarification.